README.md

```diff
@@ -1,6 +1,160 @@
-<<<<<<< HEAD
-# webpack
-An Exercise to setup a Frontend development Environment using Webpack
-=======
-# webpack
->>>>>>> 1e3f6b863829ffcc4f663cdb23af81494ccbd57d
+<a name="readme-top"></a>
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍